src/InfoboxThemeData.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+import {Model} from './model';
+
+export class InfoboxThemeData extends Model {
+
+	constructor(properties = {}) {
+		super();
+
+		this.items = null;
```

Comment:
I don't think the properties on theme data will be suitable for an array. They're all just key value pairs, so an object would be a much more suitable data structure.